templates/sc-swiper-card-product.php

```diff
@@ -11,10 +11,11 @@
  * @version  5.4.0
  *
  * Product Slider Shortcode
- * [bs-swiper-card-product order="DESC" orderby="date" posts="12" category="theme, child-themes, free, plugins"]
+ * [bs-swiper-card-product order="DESC" orderby="date" posts="12" category="theme, child-themes, free, plugins" featured="false" oos="true"]
  *
  * Optional:
- * featured="true" Shortcode it will pull featured products.
+ * featured="true" will pull featured products.
+ * oos="false" will hide out of stock products.
```

Comment:
We were trying to keep it short like the other attributes, but we understand how it can be confusing. If you'd like, we can change it to "outofstock" to keep it inline with WooCommerce's naming scheme.

Reply:
No, as edited in my request, `oos` is good and merged. Do you have some more ideas right now? Asking because we should release now or wait a bit.

Reply:
FYI, quickly changed `oos` into `outofstock`.  After some thought, I think it's better.

Reply:
Agreed, thanks.